_guides/how-we-do-development.md

```diff
@@ -163,6 +160,24 @@ A good pull request should:
 * describe any consequences of the work
 * update the [changelog](#tracking-changes)
 
+#### Tidying up your commits
```

Comment:
I think this section needs to come after the bit about the review guide and the examples. Or insert it as a step before the current step 7?

Reply:
Yeah, I was trying to figure out where it worked best. Have moved it into its own step as I think it's an important one to highlight for people who are new to dxw and might not have rebased / tidied up their commits much before.